README.md

```diff
@@ -184,6 +194,43 @@ platform config file.
 FW_FILE=$FW_FILE DEVICE_IP=$DEVICE_IP RTE_IP=$RTE_IP CONFIG=$CONFIG ./scripts/regression.sh
 ```
 
+## Ansible configuration
+
+> Note: Ansible runs were tested on QEMU based tests only.
+
+Setting variable $ANSIBLE_CONFIG to yes while running tests may prepare the DUT
+to execute given test suite. The use of this tool has the following
+requirements:
+
+* use of the [Run Ansible Playbook On Supported Operating
+  Systems](./lib/ansible.robot) keyword in the Suite Setup section of the
+  running test suite,
+* preparation of the IP address and port information used to connect SSH to the
```

Comment:
I know, but then I did not think of a way to set this runtime while running robot command so we hardcoded it in https://github.com/Dasharo/open-source-firmware-validation/blob/uefi-sb-release/ansible-roles/hosts#L2. Maybe it could be set while running `ansible-playbook` in https://github.com/Dasharo/open-source-firmware-validation/blob/uefi-sb-release/lib/ansible.robot#L24